Install fail, after run ./build.sh

make[2]: *** Waiting for unfinished jobs....
[ 20%] Building CXX object ReplicaSDK/CMakeFiles/ptex.dir/ptex/PLYParser.cpp.o
[ 20%] Building CXX object ReplicaSDK/CMakeFiles/ptex.dir/ptex/PTexLib.cpp.o
[ 30%] Building CXX object ReplicaSDK/CMakeFiles/ptex.dir/ptex/EGL.cpp.o
/home/yiming/workspacefordeeplearning/Facebook/Replica-Dataset/ReplicaSDK/ptex/EGL.cpp: In constructor ‘EGLCtx::EGLCtx(bool, int, bool)’:
/home/yiming/workspacefordeeplearning/Facebook/Replica-Dataset/ReplicaSDK/ptex/EGL.cpp:155:16: error: ‘GLEW_ERROR_NO_GLX_DISPLAY’ was not declared in this scope
     if (err == GLEW_ERROR_NO_GLX_DISPLAY) {
                ^
ReplicaSDK/CMakeFiles/ptex.dir/build.make:110: recipe for target 'ReplicaSDK/CMakeFiles/ptex.dir/ptex/EGL.cpp.o' failed
make[2]: *** [ReplicaSDK/CMakeFiles/ptex.dir/ptex/EGL.cpp.o] Error 1
CMakeFiles/Makefile2:89: recipe for target 'ReplicaSDK/CMakeFiles/ptex.dir/all' failed
make[1]: *** [ReplicaSDK/CMakeFiles/ptex.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2

--- a/ReplicaSDK/ptex/EGL.cpp
+++ b/ReplicaSDK/ptex/EGL.cpp
@@ -152,10 +152,13 @@ EGLCtx::EGLCtx(const bool createCtx, const int cudaDevice, const bool createSurf
 
     GLenum err = glewInit();
 
+#ifdef GLEW_ERROR_NO_GLX_DISPLAY
     if (err == GLEW_ERROR_NO_GLX_DISPLAY) {
         std::cout << "Can't initialize EGL GLEW GLX display, may crash!" << std::endl;
     }
-    else if (err != GLEW_OK) {
+    else 
+#endif
+    if (err != GLEW_OK) {
         ASSERT(false, "Can't initialize EGL, glewInit failing completely.");
     }
 

